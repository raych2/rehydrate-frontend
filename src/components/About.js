import drop from '../assets/Water_Drop_Small.png';
import stats from '../assets/Stats_small.png';
import cart from '../assets/shopping_cart_small.png';

const About = () => {
  return (
    <>
      <div className="topicSection">
        <div className="mainHeader">
          What's <div className="rectangle">Rehydrate</div>?
        </div>
        <div className="infoContainer">
          <div className="subHeader">Purpose</div>
          <div className="subText">
            Most and more areas and its communities are being affected by
            drought and heatwaves.
          </div>
        </div>
        <div className="infoContainer">
          <div className="subHeader">Why are we losing water?</div>
          <div className="subText">
            People might not know this, but it takes gallons of water to produce
            items and run operations.
          </div>
        </div>
        <div className="infoContainer">
          <div className="subHeader">Initiative</div>
          <div className="subText">
            Corporations are running high water usage without plans to deal with
            drought and heatwaves. With education, there are many actions to
            take.
          </div>
        </div>
      </div>
      <div className="topicSection">
        <div className="mainHeader">
          Why
          <div className="rectangle" id="corp">
            Corporations
          </div>
          ?
        </div>
        <div className="infoContainer">
          <div className="subHeader">Accountability</div>
          <div className="subText">
            Corporations are running high water usage without plans to deal with
            drought and heatwaves, but they have little to no plans on dealing
            with the issues and the communities.
          </div>
        </div>
        <div className="infoContainer">
          <div className="subHeader">Advocate</div>
          <div className="subText">
            Share your research from their water usage and protest to the
            corporate or company to implement better environmental policies and
            practices.
          </div>
        </div>
        <div className="infoContainer">
          <div className="subHeader">Alternatives</div>
          <div className="subText">
            Support alternative and small businesses that practice water
            conservation (such as recycling water, etc) and work on charitable
            actions related to drought and heatwaves. Supporting local
            businesses improves your community too!
          </div>
        </div>
      </div>
      <div className="topicSection" id="why">
        <div className="mainHeader">
          Why <div className="rectangle">Rehydrate</div>?
        </div>
        <div className="imgSection">
          <img className="smallImg" alt="water drop" src={drop} />
          <div className="textSection">
            <div className="subHeader">Save Water</div>
            <div className="subText">
              Educate and keep corporations accountable for water conservation
              through initiatives to prevent drought and heatwaves, as well as
              provide resources in crises.
            </div>
          </div>
        </div>
        <div className="imgSection">
          <img className="smallImg" alt="stats" src={stats} />
          <div className="textSection">
            <div className="subHeader">See Stats</div>
            <div className="subText">
              Corporations publish annual reports which include water usage of
              their headquarters and suppliers.
            </div>
          </div>
        </div>
        <div className="imgSection">
          <img className="smallImg" alt="shopping cart" src={cart} />
          <div className="textSection">
            <div className="subHeader">Shop Blue</div>
            <div className="subText">
              Support stores that are conserving water and creating community
              initiatives dealing with drought and heatwaves.
            </div>
          </div>
        </div>
      </div>
    </>
  );
};

export default About;
