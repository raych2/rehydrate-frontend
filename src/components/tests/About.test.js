import { render } from '@testing-library/react';
import About from '../About';

describe('About component', () => {
  test('renders correctly', () => {
    const { container } = render(<About />);
    expect(container).toMatchSnapshot();
  });
});
