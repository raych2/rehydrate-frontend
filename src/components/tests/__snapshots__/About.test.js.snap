// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About component renders correctly 1`] = `
<div>
  <div
    class="topicSection"
  >
    <div
      class="mainHeader"
    >
      What's 
      <div
        class="rectangle"
      >
        Rehydrate
      </div>
      ?
    </div>
    <div
      class="infoContainer"
    >
      <div
        class="subHeader"
      >
        Purpose
      </div>
      <div
        class="subText"
      >
        Most and more areas and its communities are being affected by drought and heatwaves.
      </div>
    </div>
    <div
      class="infoContainer"
    >
      <div
        class="subHeader"
      >
        Why are we losing water?
      </div>
      <div
        class="subText"
      >
        People might not know this, but it takes gallons of water to produce items and run operations.
      </div>
    </div>
    <div
      class="infoContainer"
    >
      <div
        class="subHeader"
      >
        Initiative
      </div>
      <div
        class="subText"
      >
        Corporations are running high water usage without plans to deal with drought and heatwaves. With education, there are many actions to take.
      </div>
    </div>
  </div>
  <div
    class="topicSection"
    id="corporation"
  >
    <div
      class="mainHeader"
    >
      Why
      <div
        class="rectangle"
        id="corp"
      >
        Corporations
      </div>
      ?
    </div>
    <div
      class="infoContainer"
    >
      <div
        class="subHeader"
      >
        Accountability
      </div>
      <div
        class="subText"
      >
        Corporations are running high water usage without plans to deal with drought and heatwaves, but they have little to no plans on dealing with the issues and the communities.
      </div>
    </div>
    <div
      class="infoContainer"
    >
      <div
        class="subHeader"
      >
        Advocate
      </div>
      <div
        class="subText"
      >
        Share your research from their water usage and protest to the corporate or company to implement better environmental policies and practices.
      </div>
    </div>
    <div
      class="infoContainer"
    >
      <div
        class="subHeader"
      >
        Alternatives
      </div>
      <div
        class="subText"
      >
        Support alternative and small businesses that practice water conservation (such as recycling water, etc) and work on charitable actions related to drought and heatwaves. Supporting local businesses improves your community too!
      </div>
    </div>
  </div>
  <div
    class="topicSection"
    id="why"
  >
    <div
      class="mainHeader"
    >
      Why 
      <div
        class="rectangle"
      >
        Rehydrate
      </div>
      ?
    </div>
    <div
      class="imgSection"
    >
      <img
        alt="water drop"
        class="smallImg"
        src="Water_Drop_Small.png"
      />
      <div
        class="textSection"
      >
        <div
          class="subHeader"
        >
          Save Water
        </div>
        <div
          class="subText"
        >
          Educate and keep corporations accountable for water conservation through initiatives to prevent drought and heatwaves, as well as provide resources in crises.
        </div>
      </div>
    </div>
    <div
      class="imgSection"
    >
      <img
        alt="stats"
        class="smallImg"
        src="Stats_small.png"
      />
      <div
        class="textSection"
      >
        <div
          class="subHeader"
        >
          See Stats
        </div>
        <div
          class="subText"
        >
          Corporations publish annual reports which include water usage of their headquarters and suppliers.
        </div>
      </div>
    </div>
    <div
      class="imgSection"
    >
      <img
        alt="shopping cart"
        class="smallImg"
        src="shopping_cart_small.png"
      />
      <div
        class="textSection"
      >
        <div
          class="subHeader"
        >
          Shop Blue
        </div>
        <div
          class="subText"
        >
          Support stores that are conserving water and creating community initiatives dealing with drought and heatwaves.
        </div>
      </div>
    </div>
  </div>
</div>
`;
